src/config.js

```diff
@@ -46,22 +52,24 @@ const conf = convict({
   db: {
     host: {
       doc: 'Hostname where PostgreSQL is running',
-      format: 'url',
```

Comment:
I couldn't find a good format that allows paths and URLs; then I thought this is not a place where it makes really sense to validate.